pkg/scheduler/admission/types.go

```diff
@@ -0,0 +1,15 @@
+package admission
```

Comment:
per @deads2k's request, please create an "api" subdirectory under your admission plugin's package and put types there. You'll also need a v1 directory to put the versioned types.